src/main/java/com/greenfox/dramacsoport/petclinicbackend/controllers/appUser/AppUserController.java

```diff
@@ -9,17 +9,19 @@
 import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
 
 import java.security.Principal;
 
 @RequiredArgsConstructor
 @Controller
+@RequestMapping("api/v1/user")
 public class AppUserController {
 
     private final AppUserService appUserService;
     private final Logger logger = LoggerFactory.getLogger(AppUserController.class);
 
-    @DeleteMapping("/user/delete")
+    @DeleteMapping("/delete")
```

Comment:
Just a note: this is usually not recommended; to use the method name in the path. You do not need to tell if the `GET` endpoint is a get (like `"/api/v1/pets{{get}}"`, but it is still a get).

Reply:
Okay, got it...thx